test/__snapshots__/step-function.test.ts.snap

```diff
@@ -11,13 +11,9 @@ Object {
         "States": Object {
           "return task(item)": Object {
             "End": true,
-            "Parameters": Object {
-              "FunctionName": "__REPLACED_TOKEN",
-              "Payload.$": "$.item",
-            },
-            "Resource": "arn:aws:states:::lambda:invoke",
```

Comment:
Ah, was I calling the AWS SDK integration?

Reply:
No, the "optimized", but also returning the whole response wrapper when we didn't need it. The largest issue was the `ResultPath` being a string and not an object. It turned out easier to correctly format the input than to manipulate the response.